### DISCUSSION TITLE
libtquic.so是否能编译出支持arm架构的版本？

### DISCUSSION THREAD

**Discussion by Aquints** [Tencent/tquic#381] [Q&A (问题和回答)]
我在集成libtquic.so到C/C++项目时使用的是某个模组提供的编译环境和交叉编译工具，模组架构为arm，而libtquic.so文件通过file指令查出其架构为x86_64，在相关makefile文件中添加了libtquic.so的路径后编译，交叉编译器会跳过此路径的libtquic.so并提示skipping incompatible libtquic.so，所以想问问能否编译出支持arm架构版本的libtquic.so？

**Comment by iyangsj** [comment]
@Aquints TQUIC支持编译arm架构的库
- 如果目标平台是移动端，可以参考 https://tquic.net/zh/docs/getting_started/installation 中Android/iOS平台编译说明
- 如果目标平台是PC端，最简单的方式是在目标平台编译出tquic库再使用。后续版本会提供更简单的交叉编译方法。

**Comment by justchen** [comment]
直接在arm平台上编译,不要在X86上使用交叉编译的方式.
